### ISSUE TITLE
Chrome + NVDA: Enter on a focused link fails when a focusable Shadow DOM descendant exists

### ISSUE THREAD

**Issue by woody-li** [nvaccess/nvda#20867]
### Brief summary

When using NVDA with Google Chrome, pressing Enter on a focused native <a> element may fail to activate the link if the link contains a custom element whose Shadow DOM contains a focusable descendant.

The same testcase works correctly with NVDA + Firefox and with JAWS + Chrome.

### Steps to reproduce

Code:
```html
<a href="https://example.com" target="_blank">
    <x-test></x-test>
</a>
```
```javascript
const [el] = document.getElementsByTagName('x-test');
el.attachShadow({ mode: 'open' });
el.shadowRoot.innerHTML='<span tabindex="-1">123456</span>';
```

### Actual behavior

Only change focus to inner span element.

### Expected behavior

The focused `a` element should be activated and the browser should navigate to the link target.

### NVDA type

installed copy

### NVDA version

2026.2 (2026.2.0.57664)

### Windows version

Windows 11 Pro 10.0.26100 Build 26100

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by seanbudd** [comment]
Welcome @woody-li 

We are unable to process this issue as it stands.

- This may be a duplicate of #13026 or #17845, can you confirm?
- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

Kind Regards,
NV Access Software Developers

**Comment by woody-li** [comment]
https://codepen.io/editor/Woody-lxf/pen/01a0c6a2-683d-7574-b46c-1622382724bc
